scripts/winesapos-android.sh

```diff
@@ -0,0 +1,9 @@
+echo "This may take a while to install"
+echo "installing Android"
+# DONT REMOVE THESE LINES PLEASE THIS WILL BREAK EVERYTHING
+makepkg_fn waydroid install
```

Comment:
In this `scripts/winesapos-android.sh` file, you can have:

```
yay --noconfirm -S --removemake waydroid waydroid-image-gapps waydroid-image
```

Then in the `scripts/repo/winesapos-build-repo.sh` file immediately after the `makepkg_fn ssdfs-tools` line, you can build the packages by adding these lines:

```
makepkg_fn waydroid install
makepkg_fn waydroid-image-gapps install
makepkg_fn waydroid-image
```